### DISCUSSION TITLE
getT locale outside React Component

### DISCUSSION THREAD

**Discussion by MangoLulo** [vinissimus/next-translate#937] [Q&A]
**Hello,**

First, I'd like to thank you for this library that I'm using for about a year now. My issue is the following one: 

- I have a TS file containing an array of paragraph objects, with two key : image (as an image source) and content (as string).

- I would like my paragraph objects, within the TS file, to get the translation.

I know that **_getT_** is supposed to work outside React components. Still, I can not figure out how to get the locale that **_getT_** needs to work. If anyone has an idea, it would be wonderful.

**Thanks for reading me,**

**Comment by aralroca** [comment]
Is an async function and needs the locale, you should use it like this: 

```js
export async function someHelper({ locale }) {
  const t = await getT(locale, 'common')
  const title = t('title')
}
```

**Reply by MangoLulo** [reply]
Thanks for your help, @aralroca
